### ISSUE TITLE
The accuracy of pytorch official model is 0.3% lower than full precision

### ISSUE THREAD

**Issue by talenz** [amirgholami/ZeroQ#14]
I've used the pytorch official resnet50 (https://pytorch.org/docs/stable/torchvision/models.html) in your ZeroQ, and it only gives 
75.85 (full precision is 76.13)

All I do is change the model loading line in uniform_test.py as follows

> model = ptcv_get_model(args.model, pretrained=True)
>model = torchvision.models.resnet50(pretrained=True)

Any hypermeters are allowed to be tuned for better performance?

**Comment by jiangaojie** [comment]
> I've used the pytorch official resnet50 (https://pytorch.org/docs/stable/torchvision/models.html) in your ZeroQ, and it only gives 75.85 (full precision is 76.13)
> 
> All I do is change the model loading line in uniform_test.py as follows
> 
> > model = ptcv_get_model(args.model, pretrained=True)
> > model = torchvision.models.resnet50(pretrained=True)
> 
> Any hypermeters are allowed to be tuned for better performance?

I meet  the same issue  . Do you slove this problem? Thanks
